tests/test_all.py

```diff
@@ -340,6 +340,18 @@ def test_no_orphaned_packages(container_per_test: ContainerData) -> None:
             'search-result/solvable-list/solvable[@repository="(System Packages)"]'
         )
     }
+
+    monitoring_stack_packages = {
+        "prometheus",
+        "alertmanager",
+        "blackbox_exporter",
+        "grafana",
+        "system-user-grafana",
+        "golang-github-prometheus-alertmanager",
```

Comment:
Sorry, missed this one
```suggestion
        "golang-github-prometheus-alertmanager",
        "golang-github-prometheus-prometheus",
```